docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+
+    ```c
+    static bool fill_response_item_cb(autofill_fill_response_item_h item, void *user_data)
+    {
+        char *id = NULL;
+        char *value = NULL;
+        char *presentation_text = NULL;
+
+        autofill_fill_response_item_get_id(item, &id);
+        autofill_fill_response_item_get_presentation_text(item, &presentation_text);
+        autofill_fill_response_item_get_value(item, &value);
+
+        dlog_print(DLOG_INFO, LOG_TAG, "id : %s, value : %s, presentation text : %s", id, value, presentation_text);
+
+        if (id) {
+            free(id);
+        }
+
+        if (value) {
+            free(value);
+        }
+
+        if (presentation_text) {
+            free(presentation_text);
+        }
+
+        return true;
+    }
+
+    static bool fill_response_group_cb(autofill_fill_response_group_h group_h, void *user_data)
+    {
+        autofill_fill_response_group_foreach_item(group_h, fill_response_item_cb, group_h);
+
+        return true;
+    }
+
+    static void
+    fill_response_received_cb(autofill_h ah, autofill_fill_response_h fill_response, void *data)
+    {
+        /* get fill response group count */
+        int count;
+        autofill_fill_response_get_group_count(fill_response, &count);
+        dlog_print(DLOG_INFO, LOG_TAG, "group count : %d", count);
+
+        autofill_fill_response_foreach_group(fill_response, fill_response_group_cb, &count);
+    }
+
+    /* Set callback function for receiving autofill fill response */
+    autofill_fill_response_set_received_cb(ah, fill_response_received_cb, NULL);
+    ```
+
+2. Create autofill data for each input field and view information to include several autofill data:
+
+    ```c
+    autofill_view_info_h vi_h;
+    autofill_item_h ai_id, ai_password;
+    char *app_id = NULL;
+
+    app_get_id(&app_id);
+
+    /* create autofill item for entering an user ID */
+    autofill_item_create(&ai_id);
+    autofill_item_set_autofill_hint(ai_id, AUTOFILL_HINT_ID);
+    autofill_item_set_id(ai_id, "id");
+    autofill_item_set_label(ai_id, "Enter your ID");
+    autofill_item_set_sensitive_data(ai_id, false);
+
+    /* create autofill item for entering an user password */
+    autofill_item_create(&ai_password);
+    autofill_item_set_autofill_hint(ai_password, AUTOFILL_HINT_PASSWORD);
+    autofill_item_set_id(ai_password, "password");
+    autofill_item_set_label(ai_password, "Enter your password");
+    autofill_item_set_sensitive_data(ai_password, true);
+
+    /* create autofill view info */
+    autofill_view_info_create(&vi_h);
+    autofill_view_info_set_app_id(vi_h, app_id);
+    autofill_view_info_set_view_id(vi_h, "login");
+
+    autofill_view_info_add_item(vi_h, ai_id);
+    autofill_view_info_add_item(vi_h, ai_password);
+    ```
+
+3. Request to get the data previously entered by the user and stored in an autofill service:
```

Comment:
To get the data previously entered by the user and stored in an Autofill service, request using the `autofill_fill_request()` function: